docs/docs/axes/radial/linear.mdx

```diff
@@ -38,16 +39,7 @@ The following additional configuration options are provided by the radial linear
 
 <CommonTicksAll />
 
-The scriptable context has the following form:
-
-```javascript
-{
-    chart,
-    scale,
-    index,
-    tick
-}
-```
+The scriptable context is described into [Options](../../general/options.md#tick) section.
```

Comment:
`into` does not sound right to me. @etimberg?

Reply:
I would  say `in [Options]`

Reply:
> I would say `in [Options]`

Oki! Apologize for my poor English...

Reply:
No need to apologize 😄